drivers/misc/pio_rpi_pico/pio_rpi_pico.c

```diff
@@ -14,7 +14,7 @@
 #define DT_DRV_COMPAT raspberrypi_pico_pio
 
 struct pio_rpi_pico_config {
-	PIO pio;
+	const PIO pio;
```

Comment:
`pio` is not pointer, the `const` is not needed here.

Reply:
You are right.